tsconfig.json

```
@@ -1,22 +1,22 @@
-    "forceConsistentCasingInFileNames": true
-  },
-  "include": ["src"],
-  "exclude": [
-    "node_modules",
-    "notes*"
-  ]
-}
\ No newline at end of file
+    "compilerOptions": {
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
```

Biome (2.4.14)

[error] The key esModuleInterop was already declared.

(lint/suspicious/noDuplicateObjectKeys)